setup.py

```diff
@@ -12,16 +12,18 @@
 extras_require = {
     "docs": [
         "sphinx",
+        "sphinx-gallery",
         "furo",
         "glfw",
         "jupyter-rfb>=0.4.1",  # required so ImageWidget docs show up
         "ipywidgets>=8.0.0,<9",
         "sphinx-copybutton",
         "sphinx-design",
-        "nbsphinx",
         "pandoc",
         "jupyterlab",
         "sidecar",
+        "imageio",
+        "matplotlib"
```

Comment:
??????????

Reply:
sphinx-gallery requires matplotlib, and need imageio for the examples